main/exercise/exercise.class.php

```diff
@@ -10569,4 +10585,46 @@ private function setResultDisabledGroup(FormValidator $form)
             get_lang('ShowResultsToStudents')
         );
     }
+
+
+    /**
+     * Sort the questions alphabetically given an array of ids.
+     * $orderAsc true = Sorted ascending / $orderAsc true = Sorted descending
+     * $questionsId = iid of questions in table question
+     *
+     * @param array $questionsId
+     * @param bool $orderAsc
+     *
+     * @return array
+     */
+    public function getAlfabeticalOrderQuestion(
+        $questionsId = [],
+        $orderAsc = true
+    ) {
+        if (empty($questionsId)) return $questionsId;
+        if (!is_array($questionsId)) return $questionsId;
+
+        $tableQuiz = Database::get_course_table(TABLE_QUIZ_QUESTION);
+        $jsonQuestionId = json_encode($questionsId);
+        $jsonQuestionId = str_replace(['[', ']', '"', "'"], '', $jsonQuestionId);
```

Comment:
If the goal is only to split a flat (one dimension) array with commas, then you should do something easier, like 
```
implode(',', $questionsId);
```
The current version seems less efficient.